### ISSUE TITLE
Example: In DarkMode, a blank square appears in bottom right corner while scrollingreadest's reading area is inaccessible to the orca screen reader in linux

### ISSUE THREAD

**Issue by kendell-clark** [readest/readest#932]
## Bug Description

A clear and concise description of what the current behavior is.
Please also add **screenshots** of the existing application.

**Example:**

```
In DarkMode, when scrollbar are displayed (for example on Companies page, with enough companies in the list), we see a blank square in the bottom right corner
[screenshot]
```

## Expected behavior

A clear and concise description of what the expected behavior is.

**Example:**

```
The blank square should be transparent (invisible)
```

## Technical inputs

Operating System:

Readest Version:

**Example:**

```
Operating System: macOS 14.3.1
Readest Version: 0.9.0
We are displaying custom scrollbars that disappear when the user is not scrolling. See ScrollWrapper.
Probably fixable with CSS
```
Orca can get at most of readest's web interface but not the reading area where the book text is. This is on arch linux, linux kernel 6.14.3, readest 0.9.36

**Comment by chrox** [comment]
Please fill the issue template and describe the problem clearly. If possible please upload screenshots or video recordings.

**Comment by chrox** [comment]
It would be better to introduce Orca first as it's not so well known.

**Comment by chrox** [comment]
I have searched that Orca is a screen reader for people with visual impairments, it provides alternative access to the desktop by using speech synthesis. 

And I will test the compatibility with this software.

**Comment by kendell-clark** [comment]
most of readest is accessible with the orca screen reader, except the area where the book text is displayed, and a couple of unlabeled buttons just after the title of the book.

**Comment by kendell-clark** [comment]
I might be able to get the orca people to look into this. What do you guys use on the backend? What web software? On windows it appears to be chromium, is it the same on linux?

**Comment by chrox** [comment]
The web rendering is provided by tauri framework which uses webkitgtk on Linux and webview2 on Windows and WebView on macOS, iOS and Android.

**Comment by kendell-clark** [comment]
hmm, orca should be able to get to the reading area unless a lot of custom widgets are used. In taht case accessibility needs to be added manually

**Comment by chrox** [comment]
The Orca screen reader will be supported in the next update of version 0.9.79. Please report back if there is any problem.
